### ISSUE TITLE
Changes to a file required by a test runs zero tests

### ISSUE THREAD

**Issue by sunesimonsen** [gustavnikolaj/imocha#19]
I'm testing the too out on https://github.com/sunesimonsen/chance-generators, I'm running the tool this way:

```
> imocha src
```

I'm changing this line: https://github.com/sunesimonsen/chance-generators/blob/master/src/ArrayGenerator.js#L16

It should fail this test: https://github.com/sunesimonsen/chance-generators/blob/master/src/ArrayGenerator.spec.js#L20

One thing that is special about this setup is that the tests is collocated with the implementation, so running just `mocha` doesn't find any tests. You need to run it with `mocha`.

**Comment by gustavnikolaj** [comment]
Running `imocha 'src/**/*.spec.js'` works. When running `mocha src` mocha will scan all the source files for tests - and I special case the handling of test files; if a test file is dirty, I know to run it and I don't need to go looking for files related to it.

In your case, that doesn't really work, as that causes me to not find related files from it, and just add it to the list of files I run.

This  means that what `imocha` is running in the case you describe is actually `mocha src/ArrayGenerator.js` and there's no describe blocks in there.

**Comment by gustavnikolaj** [comment]
Commenting out the following line fixes the problem: https://github.com/gustavnikolaj/imocha/blob/c2dfae43d47402faf926023e0802c9d26827d8de/lib/findTestFilesToRun.js#L31

It's basically just an optimization not looking for related files in test files. But I am not sure it's really necessary.

**Comment by gustavnikolaj** [comment]
Released 0.3.0 with the two changes referenced above. :-) It should make it run in chance-generators with no changes, and make documentation tests work as well.

**Comment by gustavnikolaj** [comment]
It felt a bit slow, so I tried to investigate a bit. I measured the test run in imocha to take 1100 ms. mocha itself reported 180 ms. But when I time mocha externally (including the preparation time before starting to execute tests (transpiling) and including the overhead of starting node, I got a result of 1300ms. So it's not actually slower. 

The calculations done on refresh in imocha took 20ms.

I guess most of the time here is spent in babel :-)
